### PR TITLE
start commiting bindata file for openshift

### PR DESCRIPTION
**Pull request by jim-minter**
among other things, committing bindata file allows other packages to vendor should they need to without having to run `go generate`

### CODE REVIEW

**Comment by jim-minter** [comment]
@jackfrancis fyi
other bindata files not currently committed include pkg/acsengine/templates.go, pkg/i18n/translations.go.
I believe that an equivalent PR for these files would solve #1068 and #1160.

File: pkg/openshift/certgen/templates/templates.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
 //go:generate go get -u github.com/go-bindata/go-bindata/...
-//go:generate go-bindata -pkg templates master/... node/...
+//go:generate go-bindata -modtime 1 -pkg templates master/... node/...
```
Comment (pweil-) [inline pkg/openshift/certgen/templates/templates.go]: why are you overriding the mod time stamp here?

Comment (jim-minter) [reply pkg/openshift/certgen/templates/templates.go]: Git doesn't persist timestamps so if `-modtime 1` is not included here it results in a lot of churn in commits.

Comment (jim-minter) [reply pkg/openshift/certgen/templates/templates.go]: (origin uses -nometadata instead, it turns out)

**Pull request merged**